mule-packager/src/main/java/org/mule/tools/api/classloader/model/util/ArtifactUtils.java

```diff
@@ -162,4 +163,11 @@ public static BundleDescriptor toBundleDescriptor(ArtifactCoordinates artifactCo
         .setClassifier(artifactCoordinates.getClassifier())
         .setType(artifactCoordinates.getType()).build();
   }
+
+  public static BundleDependency toBundleDependency(Artifact artifact) {
```

Comment:
could you please add a test for this?

Reply:
Actually, I created that but didn't use, so I will just delete that.